### PR TITLE
Fixes #7026: The package doesn't build 3.2 on solaris

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7026

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -442,6 +442,10 @@ ifeq ($(INSTALL),ginstall)
 endif
 	cd fusioninventory-agent && PERL_MM_USE_DEFAULT=1 $(INSTALL_DIR)/bin/perl Makefile.PL --default PREFIX=$(RUDDER_DIR)
 	cd fusioninventory-agent && $(MAKE_COMMAND) install DESTDIR=$(DESTDIR)
+	# bug in fusion installer that doesn't set the right owner to some files
+	chown root $(DESTDIR) opt/rudder/share/fusioninventory
```
Comment (VinceMacBuche) [inline rudder-agent/SOURCES/Makefile]: Do you change owner of $DESTDIR three times ? or have you missed a slash  between DESTDIR and the path ?

Comment (peckpeck) [inline rudder-agent/SOURCES/Makefile]: Good point :)

**Comment by peckpeck** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
Seems good ! merging!

**Pull request merged**